packages/styling/src/interfaces/ISemanticColors.ts

```diff
@@ -71,6 +71,11 @@ export interface ISemanticColors {
    */
   bodySubtext: string;
 
+  /**
+   * Color of links/anchors (<a>)
+   */
+  bodyLink: string;
```

Comment:
Why is it bodyLink, when other stuff is like inputBorder?

Reply:
"body" is sort of the synonym for all the default text stuff. So if a page was just a blob of text (like a wiki page) then all those text styles would be bodySomething.